controllers/scan_api_utils.go

```diff
@@ -0,0 +1,126 @@
+package controllers
+
+import (
+	"fmt"
+
+	"go.mondoo.com/mondoo-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	scanApiDeploymentName = "mondoo-scan-api"
+	scanApiServiceName    = "mondoo-scan-api"
+	scanApiPort           = 8080
+)
+
+var scanApiLabels = map[string]string{"app": "mondoo-scan-api"}
+
+func ScanApiDeployment(ns string, m *v1alpha1.MondooAuditConfig) appsv1.Deployment {
+	return appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      scanApiDeploymentName,
```

Comment:
Some kind of templating here, but at the very least:
```suggestion
			Name:      fmt.Sprintf("%s-scan-api", m.Name),
```